api/__tests__/routes/garbage.test.ts

```diff
@@ -32,7 +31,12 @@ describe("Garbage tests", () => {
         beforeAll(() => {
             runner.authLevel(AuthenticationLevel.SUPER_STUDENT);
         });
-        test("GET /garbage", async () => {
+
+        describe("GET /garbage with different roles", () => {
+            afterAll(() => {
+                runner.authLevel(AuthenticationLevel.SUPER_STUDENT);
+            });
```

Comment:
Dit voelt verkeerd aan. 

Elke test is zelf verantwoordelijk voor de correcte instelling van de authenticatieniveau. 
Onze testen moeten zo goed als mogelijk decoupled zijn.
Ik stel voor om in plaats van deze constructie een `beforeEach` te gebruiken om de verwachtte auth niveau te zetten voor de test suite. Testen die toch een andere auth niveau nodig hebben kunnen dit zelf aanpassen.